definitions/ext-rf_scanner/definition.yml

```diff
@@ -0,0 +1,51 @@
+#  [mandatory] The domain of an entity, default to EXT unless otherwise indicated by the maintainers.
+domain: EXT
+# [mandatory] The type of an entity. Must be unique within its domain, but may be duplicated between domains.
+# Please add a brief description of what your type is here on the definition file.
+type: RF_SCANNER
+
+# Entity definitions without this section are synthesized based on rules defined elsewhere.
+synthesis:
+  # [mandatory] The name of a telemetry attribute that will be used as the id of the entity, so it needs to be unique,
+  # it can be the same field used for the name or not.
+  identifier: macAddress
```

Comment:
This macAddress is that of the RF_SCANNER device itself, right?

Just to double check because we had this issue with EXT-ROUTER where we had a similar field that was getting values from other devices in the network.

Reply:
Yes, this is the macAddress of the RF_SCANNER.